### ISSUE TITLE
Could not load type System.Security.Cryptography.HMACRIPEMD160  from assembly  

### ISSUE THREAD

**Issue by aghilimc** [sshnet/SSH.NET#554]
how can i solve this error?

i have main azure function class library .net core 2.2 and i have other library class to work with sftp  is FrameworkVersion v4.6.1 .

my code is:
```
   using (var client = new SftpClient(Target.Host, Convert.ToInt32(Target.Port), Target.account_name, Target.account_key))
                {
                    client.Connect();
                    client.CreateDirectory(folderPath);
                    client.Disconnect();
                }
```

and i got this error:

`Could not load type System.Security.Cryptography.HMACRIPEMD160  from assembly  mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.`

**Comment by dtotzke** [comment]
See answer in issue https://github.com/sshnet/SSH.NET/issues/551 

This is targeting net40 which is not compatible with Core. This is a drag.

**Comment by Rob-Hague** [comment]
I have not tested it, but I am confident that this is fixed in 2023.0.0 (by virtue of updated target framework support).
